### DISCUSSION TITLE
Example-Project-Rollup Error

### DISCUSSION THREAD

**Discussion by tpfleging** [adobe/spectrum-web-components#2601] [General]
I've been trying to use the Rollup example project but haven't been successful yet.

When I execute "npm run start" I get the following error

```
> example-project-rollup@0.6.3 start
> concurrently --kill-others --names tsc,wds "npm run tsc:watch" "wds --config wds.config.js"

[wds] /bin/sh: wds: command not found
[wds] wds --config wds.config.js exited with code 127
--> Sending SIGTERM to other processes..
[tsc] npm run tsc:watch exited with code SIGTERM
npm ERR! Lifecycle script `start` failed with error: 
npm ERR! Error: command failed 
npm ERR!   in workspace: example-project-rollup@0.6.3 
```

Any ideas on what I'm doing wrong?

**Comment by Westbrook** [comment]
Oh, looks like we don't usually run the project outside of the main library, and there are a couple of dependencies missing.

If you `npm add @web/test-runner -D` and then attempt that command do you have better luck?

**Reply by tpfleging** [reply]
When I run that `npm add @web/test-runner -D` I get this error:

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: @adobe/spectrum-web-components@0.0.9
npm ERR! Found: @custom-elements-manifest/analyzer@0.6.3
npm ERR! node_modules/@custom-elements-manifest/analyzer
npm ERR!   dev @custom-elements-manifest/analyzer@"^0.6.3" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @custom-elements-manifest/analyzer@"^0.4 || ^0.5" from cem-plugin-module-file-extensions@0.0.5
npm ERR! node_modules/cem-plugin-module-file-extensions
npm ERR!   dev cem-plugin-module-file-extensions@"^0.0.5" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 

```

**Reply by Westbrook** [reply]
So, you're running this from within the project. SWC is build with `yarn`, so you would be better off using it to install dependencies. If you install with `yarn` and then run the command, you shouldn't need to install anything new...

**Reply by tpfleging** [reply]
Thanks Westbrook.  Sorry it may be a dumb question but what do I install with yarn?

**Reply by Westbrook** [reply]
If you run `yarn` it's the same as `yarn install` and it will install all the dependencies of the project. NPM isn't currently configured to run the project.
